easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -168,7 +168,9 @@ def configure_step(self):
 
         if comp_fam == toolchain.INTELCOMP:
             # set preprocessor command (-E to stop after preprocessing, -C to preserve comments)
-            cpp = "%s -E -C" % os.getenv('CC')
+            # preserving comments breaks LAXlib in QE 6.6 when stdc-predef.h is present in the system
+            # cpp = "%s -E -C" % os.getenv('CC')
```

Comment:
Hmm, I think there was a good reason why we had `-C` enabled here...

Reply:
I suppose I'm missing something, but wouldn't it only matter when someone is actually looking at the preprocessed source/headers? we only keep executables in the QE install dir...

(I do usually build everything with extra debug info in order to have more information in tracebacks or when profiling with `perf`, I suppose this affects the source code that ends up in that debug info, but even then, the comments aren't necessary, I can always refer to the original source code :) )

Reply:
Yeah, I don't see why preserving comments would actually be required, that doesn't make much sense...

It was added in https://github.com/easybuilders/easybuild-easyblocks/commit/92aeb206db579afdddde9d44d4269c81c824492d, but without clear motivation.

I guess it was done because it makes it easier to navigate the code after pre-processing, which is helpful in case something went wrong during compilation?

So maybe we should only drop the use of `-C` for QE versions >= 6.6?